### PR TITLE
Detect tests framework

### LINKED ISSUE #807: import assumes framework:beakerlib
`tmt tests import` blindly assumes `framework:beakerlib`.
It should detect if beakerlib is used  in the test and set it only in such case. 
Default framework is `shell` (by documentation)

### PR DESCRIPTION
**Pull request by adka1408**
Resolves #807.

### CODE REVIEW

File: tmt/convert.py [hunk 1]
```diff
@@ -243,6 +243,21 @@ def read(path, makefile, nitrate, purpose, disabled, types):
             makefile, flags=re.MULTILINE)
         makefile = re.sub('.*rhts-lint.*', '', makefile)
 
+        # Detect framework
+        test_script = \
+            re.search(r'^run:.*\n\t\.\/(.*)$', makefile, re.M).group(1)
```
Comment (FrNecas) [inline tmt/convert.py:248]: What if there is no match? `re.search` would return None which would result in a traceback. Same below.

**Review by psss — APPROVED** [review]
Looks good. I just moved the code a bit lower in 5597e61 to prevent parsing the test script twice.

**Review by FrNecas — APPROVED** [review]
Makes sense, didn't notice that the test path was already parsed below, good simplification.

**Comment by psss** [comment]
Tests passed, the rest are known failures, merging.

**Pull request merged**